Bookish/Bookish.DataAccess/NewCopy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bookish.DataAccess
+{
+    public class NewCopy
+    {
+        public int bookId { get; set; }
```

Comment:
You're using copyId in some places and bookId in others I think. I'd recommend sticking with one, otherwise it gets confusing 🙂